overrides/assets/stylesheets/home.css

```
@@ -47,7 +47,10 @@
   background-size: 300% 100%;
   opacity: 0.4;
   z-index: -1;
-  padding: 20px;
+  padding : 20px;
+}
+.animated-gradient img{
+display: block !important;

```

nit: let's fix the formatting up here so it's indented and spaced like the rest of the file:

```suggestion
  padding: 20px;
}
.animated-gradient img {
  display: block !important;
```